### ISSUE TITLE
Condition to pull from subscribed playlist

### ISSUE THREAD

**Issue by Noggog** [simon-weber/Autoplaylists-for-Google-Music#132]
Is it possible to have autoplaylists pull from a playlist that I'm subscribed to but haven't created myself?  Right now the dropdown only lists my own playlists.

It would be useful for making roadtrip playlists where two people could make their own playlists, and then have autoplaylists merge them together.

**Comment by simon-weber** [comment]
I like it. https://github.com/simon-weber/Autoplaylists-for-Google-Music/issues/119 would need to be resolved first, though.

**Comment by SyntheticDave** [comment]
This would be really useful for me. I'd like to be able to create playlists composed of my own playlists and those created by others (basically to replicate Spotify's collaborative playlists that GPM is sorely missing).

**Comment by simon-weber** [comment]
Yeah, this is still something I'd like to figure out. I still haven't found any way to get access to full metadata for non-library songs, though.
